docs/api.rst

```diff
@@ -1307,7 +1307,7 @@ any consumers before it is terminated.
    :param string group_name: The name of the consumer group
    :param string instance: The ID of the consumer instance
 
-   :query timeout: The number of milliseconds for the underlying client library poll(timeout) request to fetch the records. Default to 5000ms.
+   :query timeout: The number of milliseconds for the underlying client library poll(timeout) request to fetch the records. Default is undefined. This parameter is used only if it's smaller than `consumer.request.timeout.ms` defined during consumer instance creation. If you didn't define any during consumer instance creation - it is ignored.
```

Comment:
```
If you didn't define any during consumer instance creation - it is ignored.
```
If you didn't define any during consumer instance creation, the one in the proxy config is taken (or its default value)